docs/aliases.rst

```diff
@@ -93,6 +93,13 @@ the ``source-foreign`` alias where the ``shell`` argument is automatically set
 to ``bash``.
 
 
+``source-zsh``
+====================
```

Comment:
```suggestion
``source-zsh``
==============
```
small nit but rst wants the number of characters in an topic marker to equal the number of characters in the topic name